app/src/Components/Project/ProjectFullDetails.tsx

```diff
@@ -2,13 +2,15 @@ import React, { FC } from 'react'
 import styled from 'styled-components/native'
 import ProjectHeading from './ProjectHeading'
 import ProjectInterestButton from './ProjectInterestButton'
+import ProjectVideo from '../Video'
```

Comment:
Small thing, but as we usually name the imported module the same as its original name (unless there's a reason to name it differently), might be better to have this as `import Video from '../Video'` and rename it as `Video` on line 93 as well.  That way it's clearer to someone else looking at the code that this is a reusable `Video` component.